dask/dataframe/rolling.py

```diff
@@ -479,6 +480,13 @@ def __init__(
         # See https://github.com/pandas-dev/pandas/issues/15969
         self._win_type = None if isinstance(self.window, int) else "freq"
 
+        if self.axis in ("index", 1, "rows"):
```

Comment:
I think there is a utility somewhere that converts this to an integer, not super important though

Reply:
Hrm, I found [validate_axis](https://github.com/dask/dask/blob/31b26dfa4bf15f23a7070155ac67e6ba285de76f/dask/array/utils.py#L497), but that also requires ndim, which I suppose we could get from `window` but that seems awkward. 

Then [`DataFrame._validate_axis` ](https://github.com/dask/dask/blob/31b26dfa4bf15f23a7070155ac67e6ba285de76f/dask/dataframe/core.py#L4225) class method which would either need importing or used from `window` as well. 

Happy to use either, but as you said it wasn't important, thought to present the two options I found before using either.

Reply:
Yeah agree, just keep it this way